### PR TITLE
Adding pro manual french

### PR DESCRIPTION
**Pull request by octaveauger**
This is the guide to using Pro without integration, intended as a placeholder until we properly launch Basic; at which point we'll complement / update the relevant parts. It's only in French for the time being, and will be linked from the inbound emails.

@harrison @jamesshedden ok from your side? Please note that the changes to html-document are the same as what's currently on dev (enabling the possibility to have pages without an English locale equivalent).
@adeclosset fyi - feel free to modify the copy
@jutta-frieden fyi - you might want to do the same for Germany later on

Screenshots from a couple of pages:
image
image
image

### CODE REVIEW

**Comment by adeclosset** [comment]
@octaveauger made some tweaks, mostly on Title and Setup page. Just need to update "Foire aux questions" => "Tutoriel".

File: app/components/html-document/html-document.js [hunk 1]
```diff
@@ -15,16 +15,16 @@ import { buildSchemaDotOrgForOrganization } from '../../helpers/schema-dot-org/s
 import { PropTypes } from '../../helpers/prop-types/prop-types';
 
 function relAlternateLinks(root, locales) {
-  var defaultPath = locales[defaultLocale].path;
+  var defaultPath = locales[defaultLocale];
```
Comment (harrison) [inline app/components/html-document/html-document.js]: Shouldn't be showing up, merge dev in?

File: app/messages/en.js [hunk 2]
```diff
@@ -176,7 +176,7 @@ export default {
     title: 'FAQ for merchants - Signing up',
     nav_title: 'Signing up',
     description: '',
-  },
+  },
```
Comment (harrison) [inline app/messages/en.js]: extra spaces :) you should get the editorconfig package for sublime, I'm happy to help with this!

File: app/pages/faq/pro-dashboard/faq-pro-dashboard-page.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import Page from '../../../components/page/page';
+import Message from '../../../components/message/message';
+import Link from '../../../components/link/link';
+import IfLinkExists from '../../../components/if-link-exists/if-link-exists';
+import FaqHeader from '../faq-header';
+import FaqSidebar from '../faq-sidebar';
+import { PropTypes } from '../../../helpers/prop-types/prop-types';
+
+export default class FaqProDashboardPage extends React.Component {
+  displayName = 'FaqProDashboardPage';
```
Comment (harrison) [inline app/pages/faq/pro-dashboard/faq-pro-dashboard-page.js]: When you're all done with the content I want to refactor this so we can re-use the same page for all faq pages, it's almost identical to faq customer and merchant. I'm happy to sort it out!

**Comment by octaveauger** [comment]
@harrison CircleCI fixed, you can do your refactor of the FAQ header and page structure and then we'll be ready to merge

**Comment by harrison** [comment]
@octaveauger @adeclosset done the refactor, wanna give it a quick check and merge?

**Comment by octaveauger** [comment]
@harrison we want to have the main title "Foire aux questions" different depending on the FAQ type. It looks like prop is faq so we can't tailor it. Could you let me know if that's possible and how? Ideally we'd have faq_merchant.title, faq_pro_dashboard.title or something alike.
image

File: app/pages/faq/pro-dashboard/faq-pro-dashboard-page.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+  };
+
+  render() {
+    return (
+      <Page isInverted={false}>
+        <FaqPage
+        category='faq.pro_dashboard'
+        title={(<Message pointer='faq.title' />)}
```
Comment (harrison) [inline app/pages/faq/pro-dashboard/faq-pro-dashboard-page.js]: @octaveauger you can set the header title here to something like `title={(<Message pointer='faq_pro_dashboard.title' />)}`

**Comment by harrison** [comment]
@octaveauger yup possible, added a line comment to where it's set

**Comment by octaveauger** [comment]
@harrison good to go, I made my change

**Pull request merged**